util/system/path.go

```diff
@@ -13,10 +13,8 @@ import (
 // ':' character .
 const DefaultPathEnvUnix = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 
-// DefaultPathEnvWindows is windows style list of directories to search for
-// executables. Each directory is separated from the next by a colon
-// ';' character .
-const DefaultPathEnvWindows = "c:\\Windows\\System32;c:\\Windows"
+// DefaultPathEnvWindows is empty to allow windows to set the PATH when it runs
+const DefaultPathEnvWindows = ""
```

Comment:
nit: so this ensures backward compatibility with classic `docker build`, but on the downside, you can't do something like this:

```dockerfile
ENV PATH "$PATH;C:\\add\\my\\path"
```

You end up with `PATH=;C:\\add\\my\\path`, and the one coming from the registry is now ignored. We therefore end up with some inconsistencies.